.github/workflows/daily_modin_precommit.yml

```diff
@@ -242,8 +242,8 @@ jobs:
             .tox/.coverage
             .tox/coverage.xml
 
-  test-enable-cte-optimization:
-    name: Test Enable CTE Optimization modin-${{ matrix.os.download_name }}-${{ matrix.python-version }}-${{ matrix.cloud-provider }}
+  test-disable-cte-optimization:
```

Comment:
I'm not sure I'm getting it. How come we're disabling the CTE optimization here and inn many other places, but still decreasing the sql counts?

Reply:
This is a new test with disable cte optimization. In this test, we actually ignore sql count check (https://github.com/snowflakedb/snowpark-python/blob/37fe60335c17e0594831bb14632c7391fd9561f7/.github/workflows/daily_modin_precommit.yml#L298)